passage/src/main/java/id/passage/android/CurrentUser.kt

```
@@ -6,98 +6,32 @@ import id.passage.android.exceptions.AddDevicePasskeyException
-                email = modelsCurrentUser.email,
-                emailVerified = modelsCurrentUser.emailVerified,
-                id = modelsCurrentUser.id,
-                lastLoginAt = modelsCurrentUser.lastLoginAt,
-                loginCount = modelsCurrentUser.loginCount,
-                phone = modelsCurrentUser.phone,
-                phoneVerified = modelsCurrentUser.phoneVerified,
-                status = modelsCurrentUser.status,
-                updatedAt = modelsCurrentUser.updatedAt,
-                userMetadata = modelsCurrentUser.userMetadata,
-                webauthn = modelsCurrentUser.webauthn,
-                webauthnDevices = modelsCurrentUser.webauthnDevices,
-                webauthnTypes = modelsCurrentUser.webauthnTypes,
-            )
-        }
-
-        internal fun convertToPassageUser(modelsUser: User): PassageUser {
-            return PassageUser(
-                email = modelsUser.email,
-                emailVerified = modelsUser.emailVerified,
-                id = modelsUser.id,
-                phone = modelsUser.phone,
-                phoneVerified = modelsUser.phoneVerified,
-                status = modelsUser.status,
-                userMetadata = modelsUser.userMetadata,
-                webauthn = modelsUser.webauthn,
-                webauthnTypes = modelsUser.webauthnTypes,
-            )
+    /**
+     * Get Current User
+     *
+     * Returns an instance of PassageUser, which represents an authenticated Passage user.
+     * The PassageUser class has methods that can be used to retrieve data on the current user
+     * which require authentication.
+     * @return PassageUser?
+     * @throws PassageUserException
+     */
+    suspend fun userInfo(): CurrentUserInfo? {
+        val currentUserAPI = CurrentuserAPI(Passage.BASE_PATH)
+        try {
+            return currentUserAPI.getCurrentuser(Passage.appId).user
+        } catch (e: Exception) {
+            return null
```

We should throw an exception here.